src/com/hillel/course2/lesson9/homework/Validator.java

```diff
@@ -0,0 +1,38 @@
+package com.hillel.course2.lesson9.homework;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Validator {
+    public static boolean validateName(String name) {
+        String regex = "^[a-zA-Zа-яА-ЯіІїЇєЄ ]{3,256}$";
```

Comment:
all regexs and patterns can be defined on the class level as static variables, as we don't need to define them each time we call the method